earth_enterprise/src/server/mod_fdb/conf.d/security.conf

```diff
@@ -49,6 +49,6 @@ Header set Expires 0
 AddDefaultCharset on
 
 # Limit SSL protocols.
-SSLProtocol -ALL +SSLv3 +TLSv1
+#SSLProtocol -ALL +SSLv3 +TLSv1
```

Comment:
We should provide an explanation as to why this file was changed or remove the settings from the file. I think it's confusing to keep them here, honestly, so I'd vote to remove it.